View/Components/ID3DurationPropertyConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Windows.Data;
+
+namespace View.Components
+{
+    class ID3DurationPropertyConverter : IValueConverter
```

Comment:
Want waarom hebben we dit precies nodig?

Reply:
het afkappen van de milliseconden van de duration